README.md

```diff
@@ -312,6 +319,10 @@ To contribute, please check out our contribution guide [here](https://github.com
 
 ## Others
 
+### Standard Template Library
```

Comment:
Add Standard Template Library in the correct alphabetical position under Others.